lineapy/graph_reader/artifact_collection.py

```diff
@@ -338,16 +357,40 @@ def _compose_module(
                 if len(line.lstrip(" ").rstrip(",")) > 0
             ]
 
-        module_input_parameters_body = (
-            ",".join(input_parameters_body)
-            if len(input_parameters_body) <= 1
-            else "\n"
-            + f",\n{indentation_block}".join(input_parameters_body)
-            + ",\n"
-        )
+        module_input_parameters = [
+            InputVariable(
+                variable_name=x.split("=")[0].strip(" "),
+                value=x.split("=")[1].strip(" "),
+            )
+            for x in input_parameters_body
+        ]
+
+        if len(module_input_parameters) == 0:
+            module_input_parameters_body = ""
+            parser_input_parameters = ""
```

Comment:
since all instances of module_input_parameters_body and parser_input_parameters are inside an if condition, there might be an issue where it says var not defined (on first run maybe? not sure when). i recommend that you pull out these two lines outside and only have overrides to it if the length !=0 inside an if block.